pyproject.toml

```diff
@@ -18,6 +18,16 @@ select = [
     "B"
 ]
 
+[tool.codespell]
+# note: pre-commit passes explicit lists of files here, which this skip file list doesn't override -
+# this is only to allow you to run codespell interactively
+skip = "./.git,./build,./python/_skbuild/,.*egg-info.*,./.mypy_cache,./pyproject.toml,./benchmarks/utilities/cxxopts.hpp"
+# ignore short words, and typename parameters like OffsetT
+ignore-regex = "\\b(.{1,4}|[A-Z]\\w*T)\\b"
+ignore-words-list = "thirdparty,couldn"
```

Comment:
```suggestion
```

I don't see either `thirdparty` or `couldn` in the typos found by `codespell`... is this necessary? If it's not, please remove it.